src/App.css

```diff
@@ -1,33 +0,0 @@
-.App {
```

Comment:
As I can see, this file is empty. If so you don't need this file. Just delete it.